x/concentrated-liquidity/incentives_test.go

```diff
@@ -740,6 +741,87 @@ func (s *KeeperTestSuite) TestUpdateUptimeAccumulatorsToNow() {
 		expectedIncentiveRecords []types.IncentiveRecord
 		expectedError            error
 	}
+
+	validateResult := func(ctx sdk.Context, err error, tc updateAccumToNow, balancerPoolId, poolId uint64, initUptimeAccumValues []sdk.DecCoins, qualifyingBalancerLiquidity sdk.Dec, qualifyingLiquidity sdk.Dec) {
```

Comment:
Note to reviewer all this logic for `validateResult` is copied to avoid duplicating boilerplate for the new `UpdateGivenPoolUptimeAccumulatorsToNow` function